package.json

```diff
@@ -9,14 +9,17 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "dotenv": "^16.4.5",
```

Comment:
PRAISE: Nice job setting up the project, and setting up the dependencies!

In Sprint Planning, we talked about using Bootstrap and React Bootstrap.

QUESTION: Since you're working on the back end and database this week, is it cool if I just install all the Bootstrap and React Bootstrap dependencies in my pull request for this sprint?

Reply:
Yes, just make sure you are using a different branch.